pkg/components/cluster-autoscaler/component.go

```diff
@@ -301,23 +300,23 @@ func (c *component) validatePacket(diagnostics hcl.Diagnostics) hcl.Diagnostics
 func (c *component) RenderManifests() (map[string]string, error) {
 	helmChart, err := util.LoadChartFromAssets(fmt.Sprintf("/components/%s", name))
 	if err != nil {
-		return nil, errors.Wrap(err, "load chart from assets")
+		return nil, fmt.Errorf("load chart from assets: %w", err)
 	}
 
 	if c.Provider == "packet" {
 		cl, err := packngo.NewClient()
 		if err != nil {
-			return nil, errors.Wrap(err, "create packet API client")
+			return nil, fmt.Errorf("create packet API client: %w", err)
 		}
 
 		devices, _, err := cl.Devices.List(c.Packet.ProjectID, nil)
 		if err != nil {
-			return nil, errors.Wrapf(err, "listing devices in project %q", c.Packet.ProjectID)
+			return nil, fmt.Errorf("listing devices in project '%q', got: %w", c.Packet.ProjectID, err)
```

Comment:
`%q` adds quotes automatically, there is no need to add them manually.

```suggestion
			return nil, fmt.Errorf("listing devices in project %q, got: %w", c.Packet.ProjectID, err)
```